flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumer.java

```diff
@@ -101,7 +101,11 @@
  * source or a downstream operator.
  *
  * @param <T> the type of data emitted
+ * @deprecated This consumer based on the Kinesis Consumer Library (KCL) has been deprecated, a jira
+ *     to create an equivalent consumer based on AWS SDK for Java 2.x has been raised in FLINK-25729
+ *     and planned for Flink 1.16.
```

Comment:
Please remove the 1.16 reference, this is captured in the Jira

Reply:
IMO we shouldn't deprecate something when at the time of deprecation there is no alternative. Effectively you're forcing 1.15 users to use deprecated APIs.

Reply:
I agree

Reply:
I couldn't agree more